Tests/Nimble-SnapshotsTests/Nimble_SnapshotsTests.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+@testable import Nimble_Snapshots
+
+final class Nimble_SnapshotsTests: XCTestCase {
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct
+        // results.
+        XCTAssertEqual(Nimble_Snapshots().text, "Hello, World!")
```

Comment:
How does this work? I can't seem to find where "Hello, World!" is set in the code

Reply:
Oh I should remove this test target. The tests are currently done in the bootstrap project

Reply:
Removed